python/cuml/test/test_fil.py

```diff
@@ -406,19 +406,18 @@ def test_lightgbm(tmp_path):
     num_round = 5
     bst = lgb.train(param, train_data, num_round)
     gbm_preds = bst.predict(X)
-
     model_path = str(os.path.join(tmp_path,
                                   'lgb.model'))
     bst.save_model(model_path)
     fm = ForestInference.load(model_path,
                               algo='TREE_REORG',
-                              output_class=False,
+                              output_class=True,
                               model_type="lightgbm")
 
     fil_preds = np.asarray(fm.predict(X))
     fil_preds = np.reshape(fil_preds, np.shape(gbm_preds))
 
-    assert np.allclose(gbm_preds, fil_preds, 1e-3)
+    assert np.allclose(np.round(gbm_preds), fil_preds, 1e-3)
```

Comment:
If `output_class==True`, FIL predictions are class labels, so there is no need for approximate comparison: exact comparison should be used.

Reply:
the np.round is to convert the xgboost prediction to labels

Reply:
Using `np.round` is OK. However, it would be better to use `np.array_equal`, as the comparison is exact:

`assert np.array_equal(np.round(gbm_preds), fil_preds)`